.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
I see, that makes sense, thanks for explaining it to me.

I am little bit concerned about robustness - if `couchdb2` changes a test name, your `skips` list will have to be updated, which can cause extra churn in your codebase. If a skipped-test is removed, you won't know that an entry from `skips` can be removed too.

Anyways, those are just my thoughts. I am sure you have spent more time thinking about this problem than I, so please take comments me with a grain of salt and feel free to ignore them :)